git_greppers/utils.sh

```
@@ -1,30 +1,300 @@
+  if [[ -z "$default_branch" ]]; then
+    default_branch="main"
+  fi
+  
+  echo "$default_branch"
+}
+
+# Function to get the current branch name
+get_current_branch() {
+  local repo_path=$1
+  
+  cd "$repo_path" || return ""
```

Shellcheck (0.10.0)

[error] Can only return 0-255. Other data should be written to stdout.

(SC2152)